File: Medium/Python/137.-Single-Number-II.py
class Solution:
    def singleNumber(self, nums: List[int]) -> int:
        a={}
        for i in range(len(nums)):
            if nums[i] not in a:
                a[nums[i]]=1
            else:
                a[nums[i]]+=1
        for j,k in a.items():
            if k==1:
                return j

File: Medium/Java/137.-Single-Number-II.java
class Solution {
    public int singleNumber(int[] nums) {
        int l = nums.length;
        Arrays.sort(nums);
        for (int i=0;i<l-3;i+=3){
            if (nums[i]!=nums[i+2]){
                return nums[i];
            }
    }
    return nums[l-1];
    }
}